assets/js/modules/thank-with-google/components/setup/SetupForm.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
@kuasha420 Within the same file, the button label should be updated from `Configure Subscribe with Google` to `Configure Thank with Google`

Reply:
Please update the tests as well.